packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -566,7 +566,7 @@ test.describe('PieInput - Component tests', () => {
                 expect(assistiveText).not.toBeVisible();
             });
 
-            test('should render the assistive text component if the prop is provided with the default variant if "status" is not provided', async ({ mount, page }) => {
+            test('should render the default variant if no status provided', async ({ mount, page }) => {
```

Comment:
Is the "Default text" inherent to the default variant? Not sure I fully understand this text